.github/workflows/update_docs.yml

```diff
@@ -0,0 +1,45 @@
+name: "Update Docs"
+on:
+  push:
+    branches:
+      - master
```

Comment:
Since we no longer have to worry about BK restriction, this could be on [release events](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release) now